packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -130,6 +123,43 @@ class PerformanceController
   /// This list is cleared and repopulated each time "Refresh" is clicked.
   List<TraceEventWrapper> allTraceEvents = [];
 
+  /// The tracking index for the first unprocessed trace event collected.
+  int nextTraceIndexToProcess = 0;
```

Comment:
nit: can these be made private or perhaps with a public getter if needed?

Reply:
done